### ISSUE TITLE
Redesign: Media edit mode

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#6202]
## Feature Description

Update design for media editor based on [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1009%3A4323).

- [x] Update selection handle and border size and shape
~-  Update slider using design system~ WILL BE DONE SEPARATELY, dependent on #4938 

| Current design | New design |
|-|-|
| ![Screen Shot 2021-02-04 at 22 30 54](https://user-images.githubusercontent.com/637548/106986170-d25be780-6738-11eb-8397-d058f7bb267c.png) | ![Screen Shot 2021-02-04 at 22 30 59](https://user-images.githubusercontent.com/637548/106986169-d25be780-6738-11eb-8e6c-f5ccb4593039.png) |

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by miina** [comment]
@barklund Looks like the [Slider component](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3355%3A13) is missing from the design system. Should we perhaps address that in a separate issue?

**Comment by miina** [comment]
Note: looks like https://github.com/google/web-stories-wp/issues/4938 is for adding the slider.

**Comment by csossi** [comment]
Verified in QA
